### ISSUE TITLE
BT-2: Create Braintrust Evaluation Dataset

### ISSUE THREAD

**Issue by EvieHwang** [EvieHwang/wm2#65]
## Braintrust Implementation - Phase 2: Test Data

**Reference:** [WM2 APF Evaluation Criteria](https://github.com/EvieHwang/wm2/blob/main/docs/WM2-APF-CRITERIA.md)
**Dependencies:** #64 (BT-1: Setup)

---

### Purpose

Create a Braintrust Dataset containing labeled test cases for WM2 classification evaluation.

### Tasks

1. **Dataset Design**
   - [ ] Decide holdout size (recommendation: 50-100 cases)
   - [ ] Stratify across all 5 categories (Pouch, Small Bin, Tote, Carton, Oversized)
   - [ ] Include edge cases with documented acceptable alternatives
   - [ ] Ensure no overlap with ChromaDB training data

2. **Dataset Creation**
   - [ ] Extract test cases from existing CSV data
   - [ ] Format as Braintrust dataset (input/expected pairs)
   - [ ] Upload to Braintrust
   - [ ] Version the dataset

3. **Dataset Structure**

```python
# Example dataset format
dataset = [
    {
        "input": {"description": "iPhone 15 Pro Max 256GB"},
        "expected": {"classification": "Tote"},
        "metadata": {"category": "electronics", "is_edge_case": False}
    },
    {
        "input": {"description": "Small USB-C cable 6 inches"},
        "expected": {"classification": "Pouch"},
        "metadata": {"category": "accessories", "is_edge_case": True, 
                    "acceptable_alternatives": ["Small Bin"]}
    }
]
```

### Key Decisions

- **Stratification:** Equal distribution across categories (not proportional to real-world)
- **Edge cases:** ~20% of dataset should be intentionally ambiguous
- **Ground truth source:** Human-validated labels (not raw CSV)

### Acceptance Criteria

- [ ] Dataset created in Braintrust
- [ ] 50+ labeled test cases
- [ ] All 5 categories represented
- [ ] Edge cases identified and documented
- [ ] Versioning enabled
- [ ] Ready to proceed to BT-3
